models/utils/setup_utils.py

```diff
@@ -0,0 +1,30 @@
+import configparser
+from pathlib import Path
+import getpass
+import sys
+
+def get_default_config_file_path():
+    config_file_path = str(Path(__file__).resolve().parent.parent.parent
```

Comment:
Minor nit but instead of 4 division operations could just do this once with the rhs operand being `'tabpy-server/tabpy_server/common/default.conf'`

Reply:
fixed, I'm not sure why I had it that way before thanks for catching that